robot_skills/src/robot_skills/robot.py

```diff
@@ -131,16 +131,30 @@ def reset(self):
                 bodypart.reset()
         return all(results.values())
 
+    def reset_all_arms(self, arm_timeout=0.0, close_gripper=True, gripper_timeout=None):
+        """
+        Reset all arms of the robot, and by default, close their grippers.
+
+        The arm is always reset, the gripper is closed if 'close_gripper' is True (which is the default).
+
+        :param arm_timeout: How long to wait for resetting the arm, by default 0.0
+        :param close_gripper: By default 'True', which closes the gripper. If False, the gripper is not activated,
+        :param gripper_timeout: If specified and 'close_gripper' holds, timeout for closing the gripper.
+            If not specified, gripper movement uses 'arm_timeout'.
+        """
+        for arm in self._arms.itervalues():
+            arm.reset(timout=arm_timeout)
+            if close_gripper:
```

Comment:
Why first reset before closing the gripper? In terms of safety I think it might be better to first close the gripper (if necessary) and then reset.

Reply:
good point, changed